lib/Runtime/Types/CMakeLists.txt

```diff
@@ -11,17 +8,17 @@ add_library (Chakra.Runtime.Types
     DynamicObjectSnapshotEnumerator.cpp
     DynamicObjectSnapshotEnumeratorWPCache.cpp
```

Comment:
@digitalinfinity Thanks! The issue was showing up with debug build. Updated the PR.